e2e/test/scenarios/models/models.cy.spec.js

```diff
@@ -162,6 +162,91 @@ describe("scenarios > models", () => {
     cy.location("pathname").should("eq", "/collection/root");
   });
 
+  it("allows to turn a native question with a long alias into a model (metabase#47584)", () => {
```

Comment:
can't this just be a pure backend test?

Reply:
like you can write a test that hits all the relevant backend endpoints and makes sure they return success responses or whatever

Reply:
it's ok to assume that if the backend works correctly (BE bug is fixed) then the FE will continue to work correctly

Reply:
You mean an API test? I actually did have one, but then removed it because it seemed redundant, but I still have it stashed. Would be easy to add or replace this test if that's preferred.

Reply:
Re-added the API test. Let me know if you prefer to drop the e2e test.